From e96f08e2cfe68757b4a0fbdf06c22246a5a9de4d Mon Sep 17 00:00:00 2001
From: Alessandro Astone <ales.astone@gmail.com>
Date: Tue, 15 Sep 2020 13:46:07 +0200
Subject: [PATCH] boringssl-compat: update for R

Change-Id: If4f9b0a067b9aa3fe094d64e46f9d11404516f16
---
 evp.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/evp.h b/evp.h
index 109add7..782a57e 100644
--- a/evp.h
+++ b/evp.h
@@ -22,8 +22,6 @@
 #define EVP_R_PUBLIC_KEY_NOT_RSA			106
 #define EVP_F_EVP_PKEY_ENCRYPT_OLD			152
 
-#define EVPerr(f,r)  (void)0
-
 int		EVP_PKEY_decrypt_old(unsigned char *dec_key,
 			const unsigned char *enc_key,int enc_key_len,
 			EVP_PKEY *private_key);
-- 
2.28.0.windows.1

